module.nix

```
@@ -0,0 +1,121 @@
+{ config, pkgs, lib, modulesPath, ... }:
+let
+  cfg = config.services.arkheon;
+  vhost = cfg.nginx;
+  hasSSL = vhost.onlySSL || vhost.enableSSL || vhost.addSSL || vhost.forceSSL;
```

Nitpick, `vhost` est uniquement défini pour ce test, tu pourrais faire :

```suggestion
  hasSSL = with cfg.nginx; onlySSL || enableSSL || addSSL || forceSSL;
```

---

Oui